karapace/kafka_rest_apis/__init__.py

```diff
@@ -284,7 +286,14 @@ def cluster_metadata(self, topics: Optional[List[str]] = None) -> dict:
                     self._cluster_metadata = self.admin_client.cluster_metadata(topics)
                 except NodeNotReadyError:
                     self.log.exception("Could not refresh cluster metadata")
-                    self.r({"message": "Kafka node not ready", "code": 500}, "application/json", 500)
+                    self.r(
+                        body={
+                            "message": "Kafka node not ready",
+                            "code": RESTErrorCodes.HTTP_INTERNAL_SERVER_ERROR.value,
```

Comment:
I avoided using `class RESTErrorCodes(int, Enum)` because that is a very leaky abstraction. Having a few more characters to type seems an okay trade off to avoid unexpected errors.